src/mlpack/tests/gmm_train.cpp

```diff
@@ -0,0 +1,182 @@
+#include<string>
```

Comment:
Can you take a look at the existing tests and adapt the style. https://github.com/mlpack/mlpack/wiki/DesignGuidelines should be helpful as well.

Reply:
Ok i'll rectify the code by following the design guideline.